src/main/java/edu/umich/ctools/sectionsUtilityTool/Friend.java

```diff
@@ -0,0 +1,317 @@
+
+		if (sslInitialized)
+		{
+			// remove ssl system settings
+			// configure JSSE system properties: 
+			// http://fusesource.com/docs/broker/5.3/security/SSL-SysProps.html
+			Properties systemProps = System.getProperties();
+			//systemProps.remove("javax.net.debug");
+			//important: http://stackoverflow.com/questions/6680416/apache-cxf-exception-in-ssl-communication-sockettimeout
+			//java.lang.System.setProperty("sun.security.ssl.allowUnsafeRenegotiation", "true");
+			systemProps.remove("javax.net.ssl.keyStoreType");
+			systemProps.remove("javax.net.ssl.trustStoreType");
+			systemProps.remove("javax.net.ssl.keyStore");
+			systemProps.remove("javax.net.ssl.keyStorePassword");
+			System.setProperties(systemProps);
+
+			sslInitialized = false;
+		}
+	}	
+
+	/*
+	 * Check if an account exists
+	 * Return a single integer with the following (It all succeeds of fails)
+	 * @param email friend account Email
+	 * @return
+	 * -1 = bad e-mail address
+	 * 0 = no friend account
+	 * 1 = friend account exists 
+	 *
+	 */
+	public static int checkAccountExist(String email) {
+		int rv = 0;	// default to be "no friend account"
+		try {
+			Xclient = new XmlRpcClient();
+			XmlRpcClientConfigImpl config = new XmlRpcClientConfigImpl();
+			config.setServerURL(new URL(friendUrl));
+			Xclient.setTransportFactory(new XmlRpcSun15HttpTransportFactory(Xclient));
+			Xclient.setConfig(config);
+			
+			Object[] params = new Object[]{new String[] {email}};
+			Object[] results = (Object[]) Xclient.execute("doAccountsExist", params);
+			// though the friend XML-RPC service supports batch call mode, 
+			// due to the ctools event model ( one "user added event" per user), 
+			// we will pass only one email address to the doAccountsExit call for now
+			// hence we should only get one call result returned
+			if (results != null && results.length == 1)
+			{
+				rv = ((Integer) results[0]).intValue();
+			}
+		}
+		catch (Exception e) {
+			M_log.warn("Friend checkAccountExist(): email address " + email + " " + e.getMessage());
+			e.printStackTrace();
+		}
+		return rv;
+	}
+
+	public static String readFile(String path, Charset encoding) throws IOException{
+		byte[] encoded = Files.readAllBytes(Paths.get(path));
+		return new String(encoded, encoding);
+	}
+	
+	
+	/*
+	 * Actually sends out an invite
+	 *
+	 * @param accountEmail friend account to be invited
+	 * @param currentUser the person doing the inviting
+	 * 
+	 * @param associated site id
+	 * @return
+	 * -1 = bad e-mail address
+	 * 0 = runtime problem
+	 * 1 = successfully sent invitation 
+	 */
+
+	//accountEmail == userToBeInvited
+	//currentUserEmail == instructorEmail
+	public static int doSendInvite(String accountEmail, 
+			String currentUserEmail, 
+			String instructorName) {
+		int rv = 0; // default to 
+		
+		try {
+			emailMessage = Friend.readFile(friendEmailFile, StandardCharsets.UTF_8);
+			emailMessage = emailMessage.replace("<instructor>", instructorName);
```

Comment:
Is there a reason this should be a case-sensitive match? If not using a case-insensitive match would help avoid frustrating typos.